tests/integration/sunrisa_integration.py

```diff
@@ -170,6 +168,7 @@ def set_lights_for_grow(message) -> None:
         "shelves": shelf_grows, 
         "grow_phases": recipe_phases,
         "is_new_recipe": is_new_recipe,
+        "recipe_name": "OG Kush",
```

Comment:
Can we save this as a variable and return it in like the other variables on line 179?